gems/enumerize/2.5/_test/test.rb

```diff
@@ -0,0 +1,7 @@
+require 'enumerize'
+
+class User
+  extend Enumerize
+
+  enumerize :gender, in: %i(man woman)
```

Comment:
I'd like to avoid using "gender" as an example of an enum. Actually, gender is not binary. It is not appropriate for this example. Could you replace it with another example, such as `role` like [the README](https://github.com/brainspec/enumerize#usage)?

Reply:
Thank you for pointing this out. As you say, this example is not appropriate.
Fixed to use "Role" as an example.